docker-compose.yml

```diff
@@ -30,6 +30,10 @@ services:
             WAIT_HOSTS: db:5432
             POSTGRESQL_HOST: db
             POSTGRESQL_PW: ${POSTGRESQL_PW}
+            SENTRY_BACKEND_DSN: https://34db819263f34c28809da045f841f045@crowd-sentry.loc.gov/2
+            SENTRY_FRONTEND_DSN: https://48ec47f15e484502a29879e40ed2e0c3@crowd-sentry.loc.gov/3
```

Comment:
I'm wondering whether these should be set at all or simply blank so we don't log every local typo to the Sentry server

Reply:
Well, if we want to keep the `raven test` step in the dockerfile... then we gotta make that conditional on these being set, I suppose. Otherwise the app container exits.

Reply:
This should be resolved now